source/scale/common-configure-mattermost-for-enterprise-search.rst

```diff
@@ -0,0 +1,43 @@
+.. meta::
+   :name: robots
+   :content: noindex
+
+:orphan:
+:nosearch:
+
+.. This page intentionally not accessible via the LHS navigation pane because it's included in other pages
+
+
+4. Set the server connection details for Elasticsearch or AWS OpenSearch:
+
+  a. Enter **Server Connection Address** for the enterprise search server you set up earlier.
+  b. (Optional) Enter **Server Username** used to access the enterprise search server. For AWS Elasticsearch, leave this field blank.
+  c. (Optional) Enter **Server Password** associated with the username. For AWS Elasticsearch, leave this field blank.
+  d. Set **Enable Cluster Sniffing** (Optional). Sniffing finds and connects to all data nodes in your cluster automatically. For AWS Elasticsearch, this field should be set to ``false``.
```

Comment:
@agnivade - Are any modifications needed to these Elasticsearch-specific instructions for AWS OpenSearch?

Reply:
I don't understand why it's said "For AWS Elasticsearch, leave this field blank". This is wrong. You can set the username/password for both. We might as well go ahead and remove "AWS Elasticsearch" altogether because AWS Elasticsearch can no longer be used with newer versions of MM.